frontend/packages/pipelines-plugin/integration-tests/features/pipelines/create-from-builder-page.feature

```diff
@@ -107,73 +107,57 @@ Feature: Create the pipeline from builder page
              Then user will be redirected to Pipeline Details page with header name "pipeline-params"
 
 
-        @regression @to-do
+        @regression
         Scenario: Deleting added task with delete icon in pipeline builder page: P-02-TC07
             Given user is at Pipeline Builder page
-             When user clicks on Add task
+             When user enters pipeline name as "pipeline-delete-task"
+              And user clicks Add task button under Tasks section
               And user searches "kn" in quick search bar
-              And user clicks on "Add" in "kn" task
+              And user clicks on Add button
               And user adds a task in series
               And user hovers over the newly added task
               And user clicks on delete icon
              Then user can see the task in series gets removed
 
 
-        @regression @to-do
+        @regression
         Scenario Outline: Create a pipeline with TektonHub task not present in cluster from pipeline builder page: P-02-TC08
             Given user is at Pipeline Builder page
              When user enters pipeline name as "<pipeline_name>"
-              And user clicks on Add task
-              And user searches and select "<task_name>" in the list of items based on the "tekton" provider in quick search bar
-              And user clicks on "Install and Add" in "<task_name>" task
-              And user should see the loading node until the installation is complete
-              And user should see the Create button enabled after installation
+              And user clicks Add task button under Tasks section
+              And user searches and select "<task_name>" in the list of items based on the "Community" provider in quick search bar
+              And user clicks on Install and add button
               And user clicks Create button on Pipeline Builder page
              Then user will be redirected to Pipeline Details page with header name "<pipeline_name>"
 
         Examples:
                   | pipeline_name | task_name |
-                  | p-task-1      | kn        |
+                  | ptask-1       | kn        |
 
 
-        @regression @to-do
+        @regression
         Scenario: Upgrade tasks that are already installed on the cluster in pipeline builder page: P-02-TC09
             Given user is at Pipeline Builder page
              When user enters pipeline name as "pipeline-client"
-              And user clicks on Add task
+              And user installs and removes "openshift-client"
```

Comment:
this should probably emphasize it is using the community version of the task, the red hat one does not offer multiple versions

Reply:
Updated